db/migrations/20200603124229_create_time_ilk_snapshot_query.sql

```diff
@@ -0,0 +1,66 @@
+-- +goose Up
+CREATE TYPE api.time_ilk_snapshot AS
+(
+    bucket_start TIMESTAMP,
+    bucket_end TIMESTAMP,
+    bucket_interval INTERVAL,
+    ilk_identifier TEXT,
+    block_number BIGINT,
+    rate NUMERIC,
+    art NUMERIC,
+    spot NUMERIC,
+    line NUMERIC,
+    dust NUMERIC,
+    chop NUMERIC,
+    lump NUMERIC,
+    flip TEXT,
+    rho NUMERIC,
+    duty NUMERIC,
+    pip TEXT,
+    mat NUMERIC,
+    created TIMESTAMP,
+    updated TIMESTAMP
+);
+
+CREATE FUNCTION api.time_ilk_snapshots(ilk_identifier TEXT, bucket_start TIMESTAMP, bucket_end TIMESTAMP, bucket_interval INTERVAL)
```

Comment:
Might we want to add some defaults here, perhaps for the `bucket_interval`?

Reply:
Added a 1 day default.